back/app/models/user.rb

```diff
@@ -354,7 +354,7 @@ def in_any_groups?(groups)
   def generate_slug
     return if slug.present?
 
-    self.slug = UserSlugService.new.generate_slug(self, full_name)
+    self.slug = UserSlugService.new.generate_slug(self, full_name) if self.first_name.present?
```

Comment:
Didn't Rubocop say you don't need `self`? Do you have the Rubocop extension installed?

Reply:
I run Rubocop manually, ad didn't think to run it here. I will edit out the `self`.

Reply:
I think I mentioned it before, but just in case I didn't. I like using the Rubocop VScode extension. It's easy to use if you install ruby on your machine with all the gems.